### ISSUE TITLE
[Bug]: (Accessibility) Swaps Input Field Lacks Focus State Outline

### ISSUE THREAD

**Issue by georgewrmarshall** [MetaMask/metamask-extension#26662]
### Describe the bug

The Swaps input field on the MetaMask extension does not display an outline when it is in focus. This lack of a focus state outline affects keyboard navigation users and does not meet accessibility guidelines.

### Expected behavior

The Swaps input field should have a visible outline or similar indicator when it is focused to provide visual feedback to users, especially for those using keyboard navigation.

### Screenshots/Recordings

https://github.com/user-attachments/assets/9bf28f4e-022c-48c9-928d-7e9164aad725

<img width="968" alt="Screenshot 2024-08-30 at 9 04 23 AM" src="https://github.com/user-attachments/assets/7ba7d9c8-e9b1-45ef-b5bb-9556cf71325e">

### Steps to reproduce

1. Open the MetaMask extension in the browser.
2. Navigate to the Swaps feature.
3. Use the keyboard to tab to the input field.
4. Observe that the input field does not show any outline or focus state.

### Detection stage

development (default)

### Version

12.4.0

### Build type

Other (please specify exactly where you obtained this build in "Additional Context" section)

### Browser

Brave

### Operating system

MacOS
